docs/src/standalone/pages/vegetation/canopy_energy/canopy_energy.md

```diff
@@ -0,0 +1,125 @@
+# Canopy Energy Balance (Bigleaf)
+
+This page documents the **canopy energy** module used in ClimaLand’s vegetation standalone runs.
+The ClimaLand code can be found [here](https://github.com/CliMA/ClimaLand.jl/blob/main/src/standalone/Vegetation/canopy_energy.jl).
+
+---
+
+## Prognostic canopy temperature
+
+The canopy temperature is modeled prognostically. Its time tendency is given by
+
+```math
```

Comment:
Maybe here we can clean up the signs a bit, i.e. remove the minus sign out front, and have:
\frac{LW_n+SW_n - (\mathrm{SHF} + \mathrm{LHF})+F_\mathrm{foots}}{denom}